### PR TITLE
Mocked InvestorUid in CLI

### PR DESCRIPTION
**Pull request by fmiguelgarcia**
It adds the generation of Mocked InvestorUid to the CLI.

### CODE REVIEW

File: cryptography/src/claim_proofs/mod.rs [hunk 1]
```diff
@@ -27,3 +27,6 @@ pub fn random_claim<R: RngCore + CryptoRng>(rng: &mut R) -> (CddClaimData, Scope
         },
     )
 }
+
+pub mod mocked;
+pub mod uuid;
```
Comment (ParnianAlimi) [inline cryptography/src/claim_proofs/mod.rs:32]: These shouldn't be added to the library, but looks like you've made your decision.

**Review by ParnianAlimi — APPROVED** [review]

File: cryptography/src/claim_proofs/mod.rs (hunk 1 above)
Comment (ParnianAlimi) [reply cryptography/src/claim_proofs/mod.rs:32]: You never see a signing library have _public_ API for mocking messages or private keys.

Comment (ParnianAlimi) [reply cryptography/src/claim_proofs/mod.rs:32]: Maybe add a comment here and say these 2 modules must not be used in production code.

Comment (aafshar-poly) [reply cryptography/src/claim_proofs/mod.rs:32]: We should merge until https://github.com/PolymathNetwork/cryptography/pull/97 is merged and the testnet and mainnet branches are created. Then this change can go to the testnet and not the mainnet branch

Comment (ParnianAlimi) [reply cryptography/src/claim_proofs/mod.rs:32]: That's a good idea.

Comment (fmiguelgarcia) [reply cryptography/src/claim_proofs/mod.rs:32]: Until `testnet` and `mainnet` branches are not ready, we could just use the dependencies in Core to point to this branch.
In that way, we don't block the testnet in PolymeshCore.
WDYT?

Comment (ParnianAlimi) [reply cryptography/src/claim_proofs/mod.rs:32]: Sounds good to me.

**Review by adamdossa — APPROVED** [review]

**Comment by ParnianAlimi** [comment]
It's safe to close this PR now. This change is in `develop`.

**Pull request closed without merging**